### PR TITLE
Neighbor viz gallery

### PR DESCRIPTION
**Pull request by lidanwu**
PR for issue #2, add package to extract and visualize the neighborhood images/metadata/cell segmentation of query cells.  
inputs: 

- cell metadata with unique `cell_ID` in format of `c_[slide]_[fov]__[CellId]`.
- Either file path to `CellStatsDir` that contains per FOV level of cell label images, morphology C902 images
- Or file path to `napari-cosmx` dataset which contains stitched images for cell label, morphology and optional protein images of entire slide.

### CODE REVIEW

**Review by patrickjdanaher — COMMENTED** [review]
Code looks good; just a few comments on docs. 
I'll try running this soon.

File: code/NeighVizGallery/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Package: NeighVizGallery
+Title: An R package for extracting and visualizing the neighborhood of query cells in gallery view
+Version: 0.1.0
+Authors@R: 
+    c(person("Lidan", "Wu", email = "lwu@nanostring.com", role = c("aut", "cre")))
+Description: An R package for extracting and visualizing the neighborhood of query cells in gallery view
+License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
```
[the remark this answers is not in the document]
Comment (lidanwu) [reply code/NeighVizGallery/DESCRIPTION]: The full LICENSE is added to this package as separate file.

File: code/NeighVizGallery/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+### System requirements
+- R (>= 3.5.0)
+- UNIX, Mac or Windows
+- see DESCRIPTION for full dependencies
+
+### Demo
+See the "vignettes" folder. 
+- `tutorial.R` for example usages of functions for individual task.
+
+
```
[the remark this answers is not in the document]
Comment (lidanwu) [reply code/NeighVizGallery/README.md]: Screenshots of folder structures are added to README of this package and the blog in top directory.

**Comment by lidanwu** [comment]
@patrickjdanaher The docs are updated to address the issues you raised. Please let me know if any more edits required.

**Pull request merged**